openstack/pipeline/02configure

```diff
@@ -483,6 +488,8 @@ do
                 set -- $@ --vault && cache $@
             fi
             ;;
+        --ovs)
```

Comment:
i think this is ok but maybe we should call it --ml2-ovs

Reply:
Should we also rename `--ovn` to `--ml2-ovn` then to make this option more
"symmetric"? This would cause a behavior change though, so maybe not a good
idea.

Reply:
Ok, will do that.